tutorials-in-development/Other/install-instructions/install-instructions/setup-git-bash/setup-git-bash.md

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
Hey @garrettmw , nice work on these edits! I think that this new tutorial that focuses on just R/Rstudio looks good, but would you please put it into the /tutorials-in-development/ directory for now? This will be the best way for me to remember to give it a new syncID and a few other backend details when we go to publish it on the new website. You should be able to just move this file into a similar directory in /tutorials-in-development/ and push the commit, which will update this PR. Please ping me when that happens so that I can merge this request. Thanks!